README.md

```diff
@@ -75,6 +75,29 @@ Check npm version with `npm -v` -> This should show 9.6.7 in the console
 6. Go to `Site/meshapp/` and run an `npm install`
 7. Build the frontend -> in the same directory as step 7 run `npm run build`
 8. Return to the root directory and start the backend server / run the application -> `python manage.py runserver`
+MYSQL SETUP (windows, if you have a mac, consider switching to a windows)
+1. Download Mysql over here https://dev.mysql.com/downloads/installer/ ( i downloaded the fatter file)
+2. Do the setup and do the full download(spam next until you see the password portion) 
+3. Make sure you are not using the legacy version, select the 8.x version
+    a. Make sure you remember your passwords and username(username is by default root)
+4. Spam next until you finish and then you’re done
+Make sure you write the username and password in your.env file. With that being said,that’s it for your mysql setup, next thing we do is django.
+
+1. Open a terminal and make sure mysql is installed by typing in `mysql --v`
+    a. If the command doesn’t work it may be because mysql isn’t added to your path. The default location for mysql should be: 
+        C:\Program Files\MySQL\MySQL Server 8.0\bin\mysql.exe
+            i. Possible solution: https://stackoverflow.com/questions/5920136/mysql-is-not-recognised-as-an-internal-or-external-command-operable-program-or-b
+2. Once you have mysql downloaded, create a new database inside your mysql.
+    a. Login to mysql by using  `mysql --u root --p` and then type in the password you saved previously when prompted
+        i. Do note that root is the default username so use whatever username you used
+    b. You should be prompted into they mysql shell, and then create a database by using `CREATE DATABASE mesh;`
+    c. Confirm your database is created with `SHOW DATABASES;`
+3. Open you pip environment is opened (with `pipenv shell`)
+4. Type in python `manage.py makemigrations` 
```

Comment:
Should we be having everyone do makemigrations? My understanding is that you do makemigrations to create the actual migrations files in the migrations folder for each model, then run migrate to apply those migrations. Having each person make migration files should not be necessary, as you would only need to makemigrations if someone was making changes to existing models.

People who clone the repo should be able to just do migrate immediately assuming they created their database, as it would just use the existing migration files.

Reply:
From my experiences it doesn't seem to hurt when people do "make migrations", it does hurt if migrations aren't made. Which is why i just told them to make migrations before migrate